.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE_BUG_REPORT.md

```diff
@@ -1,10 +1,9 @@
 ---
-name: 
+name:
```

Comment:
Is this intended?

Reply:
this is possibly due to merge / linting, either way it should not be a problem.